sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/certificate/_version.py

```diff
@@ -6,4 +6,4 @@
 # Changes may cause incorrect behavior and will be lost if the code is regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.1"
+VERSION = "1.0.0"
```

Comment:
I'm wondering if we should find a way to avoid shipping this submodule version - probably doesn't matter for now - but going forward I'd be concerned it would get out-of-sync with the package version.
I think the only place this would be surfaced would be the user agent telemetry - which we would want to stay consistent with the package version.

Reply:
Yeah, I'm not sure we can address it right now with GA upcoming, and it's not super critical as Confidential Ledger and the Identity Service are still versioned together currently.